deps/ziptree.h

```diff
@@ -186,11 +186,11 @@ __##name##_ZIP_FIND(struct type *root, const keytype *key) {            \
     enum ZIP_CMP eq = (cmp)(key, &(root)->keyfield);                    \
     if(eq == ZIP_CMP_EQ) {                                              \
         return root;                                                    \
-    } else if(eq == ZIP_CMP_LESS) {                                     \
+    }                                                                   \
+    if(eq == ZIP_CMP_LESS) {                                            \
```

Comment:
Why is this change necessary? It's better to extract this to a new PR.

Reply:
This and several other small styling issues are required to satisfy the static analyzer/linter.
This is required because it is considered as "bad" to have an else if after a previous return.